src/main/target/SPRACINGH7EXTREME/SPRACINGH7EXTREME.mk

```diff
@@ -0,0 +1,13 @@
+FEATURES       += VCP ONBOARDFLASH SDCARD_SDIO
+
+EXST = yes
+
+TARGET_SRC += \
+            drivers/max7456.c \
+            drivers/accgyro/accgyro_mpu.c \
+            drivers/accgyro/accgyro_mpu6500.c \
+            drivers/accgyro/accgyro_spi_mpu6500.c \
+            drivers/compass/compass_hmc5883l.c \
+            drivers/barometer/barometer_bmp388.c \
+            drivers/vtx_rtc6705_soft_spi.c
+            
```

Comment:
On GitHub it's pointing out that the file doesn't end with a newline. Not sure if it's policy or not, but it sticks out.

Also shouldn't this be target.mk? Why is there two .mk files?

Reply:
there's two .mk files as the SPRacingH7NANO will be a variant of this, PR to follow this one. Its also why previous PRs from me allowed some build file overrides.